src/current/v23.2/install-cockroachdb-linux.md

```diff
@@ -13,18 +13,20 @@ docs_area: deploy
   <a href="install-cockroachdb-windows.html"><button id="windows" data-eventcategory="buttonClick-doc-os" data-eventaction="windows">Windows</button></a>
 </div>
 
-<p>See <a href="https://www.cockroachlabs.com/docs/releases/{{page.version.version}}" class="mac-releasenotes-download" id="mac-releasenotes-download-{{page.version.version}}" data-eventcategory="mac-releasenotes-download">Release Notes</a> for what's new in the latest release, {{ page.release_info.version }}. To upgrade to this release from an older version, see <a href="upgrade-cockroach-version.html">Cluster Upgrade</a>.</p>
-
 {% include cockroachcloud/use-cockroachcloud-instead.md %}
 
+<p>See <a href="https://www.cockroachlabs.com/docs/releases/{{page.version.version}}" class="mac-releasenotes-download" id="mac-releasenotes-download-{{page.version.version}}" data-eventcategory="mac-releasenotes-download">Release Notes</a> for what's new in the latest release, {{ page.release_info.version }}. To upgrade to this release from an older version, see <a href="upgrade-cockroach-version.html">Cluster Upgrade</a>.</p>
+
 Use one of the options below to install CockroachDB.
 
-{{site.data.alerts.callout_success}}
 To install a FIPS-compliant CockroachDB binary, refer to [Install a FIPS-compliant build of CockroachDB]({% link {{ page.version.version }}/fips.md %}).
-{{site.data.alerts.end}}
 
 CockroachDB on ARM is <b><a href="https://www.cockroachlabs.com/docs/stable/cockroachdb-feature-availability#feature-availability-phases">Generally Available</a></b> in v23.2.0 and above. For limitations specific to ARM, refer to <a href="#limitations">Limitations</a>.
 
+{{site.data.alerts.callout_success}}
+To connect to a CockroachDB cluster running any supported version of CockroachDB from your local system, you can download and install any supported version of either the SQL Shell binary or the full binary for your operating system and architecture. Refer to [Releases](https://cockroachlabs.com/docs/releases/).
```

Comment:
> To connect to a CockroachDB cluster running any supported version of CockroachDB from your local system,

I'm not sure to which cluster this is referring: is the cluster i'm connecting to a Cloud Cluster, or a CRDB binary on my local system?

Reply:
Any (Linux) cluster, including on CC -- this is in the context of self-hosted Linux installs.